### DISCUSSION TITLE
Birdseye output

### DISCUSSION THREAD

**Discussion by ragenhe** [blakeblackshear/frigate#4105] [Ask A Question] [answered]
Is there any way to output the birdseye view to an ingestible stream, rtsp maybe?

Thanks
Ragen

**Comment by NickM-27** [comment, marked as answer]
Birdseye is exported as jsmpeg, the frigate card in home assistant supports this already.

There's an open feature request for having it be an rtsp stream

**Reply by ragenhe** [reply]
Thanks you!
